spec/requests/graphql/query/enabled_dynamic_fields_spec.rb

```diff
@@ -8,30 +8,31 @@
   let(:dynamic_field) { FactoryBot.create(:dynamic_field) }
   let(:default_value) { "exampleDefaultValue" }
   let(:digital_object_type) { "item" }
+  let(:graphql_digital_object_type) { "ITEM" }
   let!(:enabled_dynamic_field) do
     FactoryBot.create(:enabled_dynamic_field, project: project, digital_object_type: digital_object_type,
       dynamic_field: dynamic_field, required: false, default_value: default_value, field_sets: [field_set])
   end
 
   include_examples 'requires user to have correct permissions for graphql request' do
-    let(:request) { graphql query(project.string_key, digital_object_type) }
+    let(:request) { graphql query(project.string_key, graphql_digital_object_type) }
   end
 
   context 'when logged in user is an project contributor' do
     before { sign_in_project_contributor to: :read_objects, project: project }
     context 'when id is valid' do
-      before { graphql query(project.string_key, enabled_dynamic_field.digital_object_type) }
+      before { graphql query(project.string_key, graphql_digital_object_type) }
       it 'returns correct response' do
         expect(response.body).to be_json_eql(%({  "enabledDynamicFields": [{
-          "digitalObjectType": "item", "type": "EnabledDynamicField", "project": { "stringKey": "#{project.string_key}" },
+          "digitalObjectType": "ITEM", "type": "EnabledDynamicField", "project": { "stringKey": "#{project.string_key}" },
```

Comment:
```suggestion
          "digitalObjectType": "#{graphql_digital_object_type}", "type": "EnabledDynamicField", "project": { "stringKey": "#{project.string_key}" },
```
Another optional suggestion.